homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,38 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+    if(sequence == 0)
+    {
+        return 0;
+    }
+    else if(sequence == 1)
+    {
+        return 1;
+    }
+
+    int i {0};
+    int j {1};
```

Comment:
What are 'i' and 'j' ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...